src/config/dynamic-theme-fixes.config

```diff
@@ -7509,6 +7508,12 @@ CSS
 .buj {
     background-image: url(//ssl.gstatic.com/ui/v1/icons/mail/rfr/density_compact_v1_1x.png) !important;
 }
+div[class*="bym"][role="navigation"] {
+    background-color: rgb(25 27 28) !important;
+}
+.byl .nZ.aBP {
+    background-color: rgba(255,255,255,0.05) !important;
```

Comment:
For this one as well.

Reply:
The closest to the current version I found the variable `--lt-color-gray-700: #2F3237;`, but still it turns out not a gray shade, but a brown one, and it looks worse than the current version. Therefore, it is better to leave it as it is.